internal/types/kruizePayload/common.go

```diff
@@ -72,6 +74,26 @@ type RecommendationTerm struct {
 		Cost        RecommendationEngineObject `json:"cost,omitempty"`
 		Performance RecommendationEngineObject `json:"performance,omitempty"`
 	} `json:"recommendation_engines,omitempty"`
+	Plots Plot `json:"plots,omitempty"`
+}
+
+type Plot struct {
+	DataPoints int                  `json:"datapoints"`
```

Comment:
```suggestion
	Plots *Plot `json:"plots,omitempty"`
}

type Plot struct {
	DataPoints int                  `json:"datapoints,omitempty"`
```

In case the env var; `plots=true` is not there for Kruize, ROS won't save any box plots json in the database.

Reply:
Changes pushed